classes/mock/generator.php

```diff
@@ -721,6 +721,7 @@ protected function getParametersSignature(\reflectionMethod $method, $forceMockC
                     $parameterCode .= ' = ' . var_export($parameter->getDefaultValue(), true);
                     break;
 
+                case $parameter->allowsNull():
```

Comment:
yes signature change, but ... we have no way to determine if it's a nullable type or not. So except if we change php core, we need a way to generate mocks... otherwise atoum will break all test with nullable type.